docs/guide/writing/style-guides.md

```diff
@@ -0,0 +1,91 @@
+# Style Guides
+
+A style guide is a set of standards for the writing and design of content, defining the style to be used in communication within a particular organization. Basically, style guides are put together to clarify [the way a group of people talk and write about the things they do](http://www.writethedocs.org/style-guide/). Think of authoring best practices.
+
+If you happen to have a background in academia or journalism, you will probably be familiar the [AP stylebook](https://en.wikipedia.org/wiki/AP_Stylebook) or the [Chicago manual of style](http://www.chicagomanualofstyle.org/book/ed17/frontmatter/toc.html). Those are great resources for writing in general, particularly for grammar and syntax, but if you're reading this page chances are you are considering style in the context of technical documentation.
+
+### Technical Document
+Style guides help you write a variety of content, such as documenting the methods of an API, or presenting a overview of complex technical concepts, or focusing on how to write particular content like user manuals, release notes, or tutorials.
+
+Other style guides often focus on the reader, helping you communicate to developers, product managers, the general public, and others. They can also focus on the language itself (tone, style, grammar). Others provide best practices on how to manage your content, version control, or publication and delivery strategies.
+
```

Comment:
I'm not sure I understand the distinction(s) that these two paragraphs make. The last sentence about content management, version control, etc also doesn't seem to have much if anything to do with style guides.

Reply:
Reordered content, made clear difference, kept the last paragraph, but made it clear that documentation organization is not what we are discussing here. I still think, however, that it is important to mention.